awswrangler/data_quality/_create.py

```diff
@@ -134,6 +136,8 @@ def update_ruleset(
         Ruleset name.
     updated_name : str
         New ruleset name if renaming an existing ruleset.
+    mode : str
+        overwrite (default) or upsert.
```

Comment:
overwrite and upsert are the only modes I can think of

Reply:
How about append?